appcues/src/debug/java/com/appcues/CompositionTestHelper.kt

```diff
@@ -125,6 +127,76 @@ public fun ComposeContainer(context: Context, stepContentJson: List<String>?, tr
     }
 }
 
+// This helper supports testing any experience JSON content, rendering the given group and step index.
+@Composable
+public fun ComposeContainer(
+    context: Context,
+    experienceJson: String,
+    groupIndex: Int,
+    stepIndex: Int,
+    animated: Boolean,
+    imageLoader: ImageLoader,
+) {
+    // set up a Koin scope for testing - for experience/trait mapping, trait registry, etc
+    val scope = AppcuesKoinContext.createAppcuesScope(context, AppcuesConfig("", ""))
+
+    // this JSON is actually a qualify response, and we'll get the first experience out of it
+    val qualifyResponse = MoshiConfiguration.moshi.adapter(QualifyResponse::class.java).fromJson(experienceJson)!!
+    val experienceResponse = qualifyResponse.experiences[0] as ExperienceResponse
+
+    // update the experience to strip out any actions, as they can rely on injected dependencies
+    // that are not available using the test context
+    val updatedExperienceResponse = experienceResponse.copy(
+        steps = experienceResponse.steps.map { stepContainerResponse ->
+            stepContainerResponse.copy(
+                children = stepContainerResponse.children.map { stepResponse ->
+                    stepResponse.copy(
+                        actions = emptyMap()
+                    )
+                },
+                actions = emptyMap()
+            )
+        },
+    )
```

Comment:
this is the simplest way to just not worry about Actions in the mapping, since they often refer to internal services and end up trying to resolve dependencies that can rely on the Android context, and that is not available in snapshot tests